internal/proto/local/controller/api/resources/scopes/v1/scope.proto

```diff
@@ -10,55 +10,65 @@ import "google/protobuf/struct.proto";
 import "controller/custom_options/v1/options.proto";
 
 message ScopeInfo {
-	// Output only. The ID of the Scope.
-	string id = 1;
+  // Output only. The ID of the Scope.
+  string id = 1;
 
-	// Output only. The type of the Scope.
-	string type = 2;
+  // Output only. The type of the Scope.
+  string type = 2;
 
-	// Output only. The name of the Scope, if any.
-	string name = 3;
+  // Output only. The name of the Scope, if any.
+  string name = 3;
 
-	// Output only. The description of the Scope, if any.
-	string description = 4;
+  // Output only. The description of the Scope, if any.
+  string description = 4;
 
-	// Output only. The ID of the parent Scope, if any. This field will be empty if this is the "global" scope.
-	string parent_scope_id = 5 [json_name="parent_scope_id"];
+  // Output only. The ID of the parent Scope, if any. This field will be empty if this is the "global" scope.
+  string parent_scope_id = 5 [json_name = "parent_scope_id"];
+
+  // Output only. The ID of the primary auth method for this scope. A primary auth method
+  // is allowed to vivify users when new accounts are created and is the source for
+  // the users account info
+  string primary_auth_method_id = 6;
```

Comment:
Please add `[json_name = "primary_auth_method_id"]`

Reply:
ScopeInfo provides info regarding a resource's parent scope.   I'm having a hard time thinking of a situation where knowing a parent scope's primary auth method would be helpful, but maybe you or @randallmorey have already identified a situation?

Reply:
Hum... I agree that it's not useful, so I'm removing it.

Reply:
ty!